src/components/menu-list/menu-list.stories.mdx

```diff
@@ -0,0 +1,167 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { MenuList, MenuListItem } from '.'
+
+<Meta title="MenuList" parameters={{ info: { disable: true }}} />
+
+# MenuList
+A set of related items that can be shown or hidden.
+
+Children can be shown or hidden by clicking their title. Useful to create simple accordion or menu patterns.
+
+## Quick Start
+Import `MenuList` and `MenuListItem` into the project.
+
+```jsx
+import { MenuList, MenuListItem } from "carbon-react/lib/components/menu-list";
+
+const MyComponent = () => (
+  <MenuList>
+    <MenuListItem>
+      Menu Item One
+    </MenuListItem>
+  </MenuList>
+);
```

Comment:
If I remember correctly we don't need this one. Just `import` is enough for this section

Reply:
I put this example in to demonstrate the semantics. Happy to remove it if you think it doesn't add any value though.

Reply:
I have removed this block of code as per the discussion in the squad channel 😄